src/shiv/pip.py

```diff
@@ -17,12 +18,25 @@ def clean_pip_env() -> Generator[None, None, None]:
     """
     require_venv = os.environ.pop(PIP_REQUIRE_VIRTUALENV, None)
 
+    distutils_cfg = Path.home() / ".pydistutils.cfg"
+    distutils_cfg_already_existed = distutils_cfg.exists()
```

Comment:
I feel like there should be some sort of warning if this already exists, though I'm not sure if you'd want to use logging, `click.echo()`, `print()` or something else entirely